src/Spain/Mews.Fiscalizations.Spain.Tests/Basics.cs

```diff
@@ -70,9 +70,9 @@ private async Task<SimplifiedInvoice> SuccessfullyPostInvoice(Client client)
 
             var response = await client.SubmitSimplifiedInvoiceAsync(model);
 
-            var responseErrorMessages = response.Invoices.Select(i => i.ErrorMessage).Flatten();
+            var responseErrorMessages = response.SuccessResult.Invoices.Select(i => i.ErrorMessage).Flatten();
```

Comment:
So then this is a null pointer if it fails. How did we improve the situation?

Reply:
There is used to be an assert which was checking whether response result is successful. However feedback was that it make test more confusing. Would you prefer to have it back?

Reply:
No, I prefer to have type safety, rather than contract check. So 2 public stacit methods - 1 success and 1 error and a private constructor. 
That way, even the user understands the usage without a need for any check.